### PR TITLE
Promisify

### PR DESCRIPTION
**Pull request by SamVerschueren**
I know you already did it, but wanted to share my implementation. I just saw you did it when I wanted to make the PR :).

Tests use `async`/`await`. Just pick the bits and pieces you like (or not) :beers:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,89 +1,64 @@
...
-				return;
+			return Promise.all([
+				fs.move(src, dest, {mkdirp: true}).then(function () {
+					return dest;
+				}),
+				fs.outputFile(info, msg).then(function () {
+					return info;
+				})
+			]);
+		})
```
Comment (sindresorhus) [inline index.js]: @kevva You should definitely cherrypick this part.

Comment (kevva) [inline index.js]: Oh, yeah. I keep forgetting using `Promise.all` instead of nesting them.

**Comment by kevva** [comment]
I think I got the bits I needed now. Thank you very much :D.

**Comment by SamVerschueren** [comment]
No problem, you can close it down if you want ;).

**Pull request closed without merging**